Dockerfile

```diff
@@ -34,13 +34,18 @@ FROM base as back-builder
 
 WORKDIR /builder
 
-# Copy distributed application's statics
-COPY --from=front-builder /builder/src/richie/static/richie /builder/src/richie/static/richie
-
 # Copy required python dependencies
 COPY setup.py setup.cfg MANIFEST.in /builder/
 COPY ./src/richie /builder/src/richie/
 
+# Copy distributed application's statics
+COPY --from=front-builder \
+    /builder/src/richie/static/richie/js/*.js \
```

Comment:
You are doing this to be sure to not have `.map` files ?

Reply:
Not specially to avoid copying the `.map` files but rather to explicitly copy all JS files and not a folder that may override existing JS files in the statics directory (as it is the case for the CSS directory).